tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/test_formula_8_4.py

```diff
@@ -0,0 +1,264 @@
+    Form8Dot4DesignAnchorageLength,
+)
+from blueprints.codes.eurocode.nen_en_1992_1_1_c2_2011.chapter_8_detailing_of_reinforcement_and_prestressing_tendons.formula_8_6 import (
+    Form8Dot6MinimumTensionAnchorage,
+)
+from blueprints.codes.eurocode.nen_en_1992_1_1_c2_2011.chapter_8_detailing_of_reinforcement_and_prestressing_tendons.formula_8_7 import (
+    Form8Dot7MinimumCompressionAnchorage,
+)
+from blueprints.validations import NegativeValueError
+
+
+class TestForm8Dot4DesignAnchorageLength:
+    """Validation for formula 8.4 from NEN-EN 1992-1-1+C2:2011."""
+
+    def test_evaluation(self) -> None:
+        """Test the evaluation of the result."""
+        # example values
+        alpha_1 = 1  # [-]
+        alpha_2 = 1  # [-]
+        alpha_3 = 1  # [-]
+        alpha_4 = 1  # [-]
+        alpha_5 = 1  # [-]
+        l_b_rqd = 450  # mm
+        l_b_min = 200  # mm
+        form_8_4 = Form8Dot4DesignAnchorageLength(
+            alpha_1=alpha_1,
+            alpha_2=alpha_2,
+            alpha_3=alpha_3,
+            alpha_4=alpha_4,
+            alpha_5=alpha_5,
+            l_b_rqd=l_b_rqd,
+            l_b_min=l_b_min,
+        )
+
+        # manually calculated result
+        manually_calculated_result = 450  # mm
+
+        assert form_8_4 == pytest.approx(expected=manually_calculated_result, rel=1e-4)
+
+    def test_evaluation_minimum(self) -> None:
+        """Test the evaluation of the result if the minimum is reached."""
+        # example values
+        alpha_1 = 1  # [-]
+        alpha_2 = 1  # [-]
+        alpha_3 = 1  # [-]
+        alpha_4 = 1  # [-]
+        alpha_5 = 1  # [-]
+        l_b_rqd = 200  # mm
+        l_b_min = 300  # mm
+        form_8_4 = Form8Dot4DesignAnchorageLength(
+            alpha_1=alpha_1,
+            alpha_2=alpha_2,
+            alpha_3=alpha_3,
+            alpha_4=alpha_4,
+            alpha_5=alpha_5,
+            l_b_rqd=l_b_rqd,
+            l_b_min=l_b_min,
+        )
+
+        # manually calculated result
+        manually_calculated_result = 300  # mm
+
+        assert form_8_4 == pytest.approx(expected=manually_calculated_result, rel=1e-4)
+
+    def test_raise_error_if_alpha_1_is_negative(self) -> None:
+        """Test that a NegativeValueError is raised if alpha_1 is negative."""
+        # example values
+        alpha_1 = -1  # [-]
+        alpha_2 = 1  # [-]
+        alpha_3 = 1  # [-]
+        alpha_4 = 1  # [-]
+        alpha_5 = 1  # [-]
+        l_b_rqd = 450  # mm
+        l_b_min = 200  # mm
+
+        with pytest.raises(NegativeValueError):
+            Form8Dot4DesignAnchorageLength(
+                alpha_1=alpha_1,
+                alpha_2=alpha_2,
+                alpha_3=alpha_3,
+                alpha_4=alpha_4,
+                alpha_5=alpha_5,
+                l_b_rqd=l_b_rqd,
+                l_b_min=l_b_min,
+            )
```

Comment:
there is a lot of repetition here (for 'raise error on negative' test), I would suggest to use: @pytest.mark.parametrize.

an example on how that works:

```python
def my_function(a, b, c):
    if a == 0 or b == 0 or c == 0:
        raise ValueError("One of the arguments is zero")

@pytest.mark.parametrize("a, b, c", [
    (1, 2, 3),
    (1, 0, 3),
    (0, 0, 3),
    (0, 0, 0)
])
def test_my_function(a, b, c):
    with pytest.raises(ValueError):
        my_function(a, b, c)          
```

@egarciamendez , @johan-tuls : we could consider using "hypothesis" (https://hypothesis.readthedocs.io/en/latest/quickstart.html) to have a more complete approach of test situation --> maybe something to discuss during the maintainers meeting, offcourse it would mean having an extra dependency